bundles/org.openhab.binding.boschshc/DEVELOPERS.md

```diff
@@ -0,0 +1,50 @@
+# For Developers
+
+## Build
+
+To only build the Bosch SHC binding code execute
+
+    mvn -pl :org.openhab.binding.boschshc install
+
+## Execute
+
+After compiling a new ``org.openhab.binding.boschshc.jar`` 
+copy it into the ``addons`` for of your openHAB test instance.
```

Comment:
Typo: Remove "for"

Reply:
Thanks @JohannesPtaszyk :) Fixed the typo!

Reply:
Thank you! Looking forward to this merge! ☺️